sqrl-examples/src/main/java/com/datasqrl/data/GenerateCreditCards.java

```diff
@@ -0,0 +1,254 @@
+package com.datasqrl.data;
+
+import com.datasqrl.cmd.AbstractGenerateCommand;
+import com.datasqrl.util.Configuration;
+import com.datasqrl.util.RandomSampler;
+import com.datasqrl.util.SerializerUtil;
+import com.datasqrl.util.WriterUtil;
+import com.github.javafaker.CreditCardType;
+import java.time.Instant;
+import java.time.temporal.ChronoUnit;
+import java.util.ArrayList;
+import java.util.Comparator;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.Set;
+import java.util.stream.Collectors;
+import java.util.stream.IntStream;
+import lombok.Value;
+import picocli.CommandLine;
+
+@CommandLine.Command(name = "creditcard", description = "Generates Credit Card data")
+public class GenerateCreditCards extends AbstractGenerateCommand {
```

Comment:
How do you feel about adding these to the creditcard example folder instead of having sqrl-examples have a class root? Examples should be reasonably self contained so they are easily reproducible for users.